src/patternfly/layouts/Level/styles.scss

```diff
@@ -0,0 +1,21 @@
+.pf-l-level {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0;
+  margin: 0;
+
+  &__group {
```

Comment:
there was a child selector here before. Do we need it in case of nested levels?

Reply:
Normally we would, but in this scenario I don't think so. Because we always want that selector to get those styles, even if it is nested.

I could also do something like this though: `& > &__group`

Reply:
nah, it's ok as is. we can change it later if we see a problem